ui/components/ui/typography/typography.stories.js

```diff
@@ -276,17 +355,39 @@ export const As = (args) => (
 );
 
 export const Margin = (args) => (
-  <Typography {...args}>
-    This Typography component has a margin of {args.margin * 4}px
-  </Typography>
+  <>
+    <BannerAlert
+      severity={SEVERITIES.WARNING}
+      title="Deprecated"
+      description="<Typography/> has been deprecated in favour of the <Text /> component"
+      actionButtonLabel="See details"
+      actionButtonProps={{
+        href: 'https://github.com/MetaMask/metamask-extension/issues/17670',
+      }}
+    />
+    <Typography {...args}>
+      This Typography component has a margin of {args.margin * 4}px
+    </Typography>
+  </>
 );
 
 Margin.args = {
   margin: 4,
 };
 
 export const BoxProps = (args) => (
-  <Typography {...args}>This uses the boxProps prop</Typography>
+  <>
+    <BannerAlert
+      severity={SEVERITIES.WARNING}
+      title="Deprecated"
+      description="<Typography/> has been deprecated in favour of the <Text /> component"
+      actionButtonLabel="See details"
+      actionButtonProps={{
+        href: 'https://github.com/MetaMask/metamask-extension/issues/17670',
+      }}
+    />
+    <Typography {...args}>This uses the boxProps prop</Typography>
+  </>
 );
 
 BoxProps.args = {
```

Comment:
Might be a bit overkill doing every story but I suppose better safe than sorry 😅

Reply:
🤣